changelog.txt

```diff
@@ -1,3 +1,8 @@
+Version 5.2.1
+==============
+Bug fix:
+- **Fixed issue with integrated windows authentication in seamless SSO hybrid environments where users were not able to sign in.** [Issue 1478](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/issues/1478)
```

Comment:
I think you can make it more exciting, as it's a pretty big deal.
JM always says to put the problem first and then the solution.

idea:
**When using integrated Windows authentication in hybrid environments, managed (cloud) users were not able to sign-in**. Now, ADAL.NET sends the correct header information to enable seamless SSO in hybrid environments.

-OR-
**When using integrated Windows authentication in hybrid environments, managed (cloud) users were not able to sign-in**. Now, managed users are able to benefit from seamless SSO in hybrid environments.

IDK..just a suggestion, what's there now is a pretty boring statement considering the impact here.

Reply:
ok, makes sense

Reply:
Super suggestion. Perhaps a link to the stuff you showed us today with a link to the additional information on SSO/Hybrid stuff